rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -541,6 +541,11 @@ def element_attribute(element, name)
           execute :getElementAttribute, id: element.ref, name: name
         end
 
+        # Backwards compatibility for w3c
+        def element_property(element, name)
+          execute :getElementAttribute, id: element, name: name
```

Comment:
Calling this atom will get an attribute instead of the property also. The intention of doing the divide of getting attribute / property is to allow users calling these to "do what I say" rather than our past approach of "do what I mean".

getting element property i don't believe is implemented in an atom. Here's what python does:

https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webelement.py#L107-L110

It falls back to essentially `return driver.execute_script('return arguments[0][arguments[1]]', element, property_name)`